### PR TITLE
feat: added plot_lagplot

### LINKED ISSUE #519: feat: Add lag-plot for time series
### Is your feature request related to a problem?

When working with time series, a lag plot is useful to see correlation between values and time. 
So when implementing this issue, the "lagging" itself must be implemented and the visualization.

### Desired solution

The user should call something like this:  ```ts.plot_lagplot()```

### Possible alternatives (optional)

_No response_

### Screenshots (optional)

_No response_

### Additional Context (optional)

_No response_

### PR DESCRIPTION
**Pull request by Gerhardsa0**
Closes #519 

### Summary of Changes

I added the visualization of the lag plot.

### CODE REVIEW

File: src/safeds/data/tabular/containers/_time_series.py [hunk 1]
```diff
@@ -857,3 +862,12 @@ def transform_column(self, name: str, transformer: Callable[[Row], Any]) -> Time
             ),
             time_name=self.time.name,
         )
+
+    def plot_lag(self, lag: int) -> Image:
```
Comment (lars-reimann) [inline src/safeds/data/tabular/containers/_time_series.py:866]: `plot_lagplot` would be more consistent with the plotting methods in `Table` (`plot_<name_of_plot>`).

Comment (Gerhardsa0) [reply src/safeds/data/tabular/containers/_time_series.py]: named it plot_lagplot

File: src/safeds/data/tabular/containers/_time_series.py [hunk 2]
```diff
@@ -857,3 +863,34 @@ def transform_column(self, name: str, transformer: Callable[[Row], Any]) -> Time
             ),
             time_name=self.time.name,
         )
+
+    def plot_lagplot(self, lag: int) -> Image:
+        """
+        Plot a lagplot for target column.
```
Comment (lars-reimann) [inline src/safeds/data/tabular/containers/_time_series.py:869]:
```suggestion
        Plot a lagplot for the target column.
```

**Review by lars-reimann — DISMISSED** [review]
Looks great, feel free to merge it in when you're done with it.

File: src/safeds/data/tabular/containers/_time_series.py [hunk 3]
```diff
@@ -857,3 +863,34 @@ def transform_column(self, name: str, transformer: Callable[[Row], Any]) -> Time
...
+
```
[the remark this answers is not in the document]
Comment (Gerhardsa0) [reply src/safeds/data/tabular/containers/_time_series.py:882]: added

**Review by lars-reimann — APPROVED** [review]

**Pull request merged**